main.ts

```diff
@@ -18,7 +18,12 @@ interface Params {
      * Git repos are chosen from Typescript-language repos based on number of stars; default is 100.
      * User repos start at the top of the list; default is all of them.
      */
-    repoCount?: number | undefined;
+     repoCount?: number | undefined;
```

Comment:
was the indentation wrong before or is it wrong now?

Reply:
Definitely wrong now D:

Reply:
```suggestion
    repoCount?: number | undefined;
```